### DISCUSSION TITLE
Music / Sound issues?

### DISCUSSION THREAD

**Discussion by awldune** [ihhub/fheroes2#5050] [Q&A]
Looking through the discussion here I don't seem to see much about sound issues. I am getting a sort of wavering or stuttering effect in the music when playing.

I'm on Windows 7 and using the gog version of Heroes 2. I can play the music OGG files fine outside of the game.

The music also sounds fine through the DOS executable, but of course that is a completely different engine!

Any ideas?

Thanks

**Comment by zenseii** [comment]
Was this ever answered or solved?

**Comment by awldune** [comment]
No, I never heard from anyone. Thanks.

**Reply by ihhub** [reply]
Hi @awldune , can you record what you hear so it might help to identify the problem?

**Reply by awldune** [reply]
Does this link work?

https://drive.google.com/file/d/1w5UFZ0e5Fn8ExjNyUyocpRM-svxK2IGd/view?usp=sharing

I play a lot of games, new and old, and I have never heard this issue.

**Reply by zenseii** [reply]
Yes, that link shows the problem. Not sure what causes it. Sounds like some interruptions. Maybe you can try the latest snapshot of fheroes2 to see if anything has changed since work has been done on the sound front? https://github.com/ihhub/fheroes2/blob/master/docs/INSTALL.md#snapshots-latest-builds

**Reply by awldune** [reply]
The issue is still there in the latest snapshot.

**Comment by awldune** [comment]
I'll have to see if I can figure out how. I have Audacity, I think it
should be able to capture the game audio.

**Reply by zenseii** [reply]
Since you have the GoG release you could also try to use the .flac audio that gog provides. There's a script that sets up the files for you since they need to be renamed.
